features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/RoomListPresenter.kt

```diff
@@ -176,27 +175,23 @@ class RoomListPresenter @Inject constructor(
     private fun securityBannerState(
         securityBannerDismissed: Boolean,
         needsSlidingSyncMigration: Boolean,
```

Comment:
Is it not because `needsSlidingSyncMigration` needs to be wrapped in a state like `securityBannerDismissed`, with `rememberUpdatedState`?